src/db.rs

```diff
@@ -467,11 +581,83 @@ mod tests {
         ];
 
         let ksetid = Uuid::new_v4().as_hyphenated().to_string();
-        let polid = 1;
-        insert_keyset(&ksetid, &keys, Some(polid))?;
-        let keyset_ids = get_keyset_ids(&ksetid)?;
+        let polid: Option<u32> = Some(1);
```

Comment:
[style] same idea, this can be simplified to: `let polid = Some(1u32);`

Reply:
Done.